.github/workflows/build.yaml

```diff
@@ -63,27 +67,23 @@ jobs:
         run: >-
           touch .env.development;
           touch .env;
-          TAG=latest docker-compose pull web || true
```

Comment:
I am not sure if I truly understand this command
`TAG=latest docker-compose pull web || true`
I interpret this as pull the web pod tagged latest, or is it set TAG=latest pull web pod?